src/index.js

```diff
@@ -49,8 +49,11 @@ function toMatchImageSnapshot(received, { customSnapshotIdentifier = '', customD
     pass = false;
   }
 
-  const message = 'Expected image to match or be a close match to snapshot.\n'
-                  + `${chalk.bold.red('See diff for details:')} ${chalk.red(result.diffOutputPath)}`;
```

Comment:
I'd be worried about the possibility of these messages getting out of sync. Might be better to use a [chalk context](https://github.com/chalk/chalk/tree/v1.1.3#chalkenabled):

```
var ctx = new chalk.constructor({enabled: !noColors});
```